code/__HELPERS/type2type.dm

```diff
@@ -20,8 +20,6 @@
 	while(i > 0)
 		var/char = copytext(hex, i, i + 1)
 		switch(char)
-			if("0")
-				//Apparently, switch works with empty statements, yay! If that doesn't work, blame me, though. -- Urist
```

Comment:
Maybe related to this removal? This would lead to it running the `else` case which returns from the proc instead of continuing.

Reply:
Ah yeah, that could very well be what's happening.